### ISSUE TITLE
Merging discord mobile repos with this repo.

### ISSUE THREAD

**Issue by isabelincorp** [catppuccin/discord#289]
Fundamentally, there are a couple of issues with discord mobile clients

- They are on End-of-life ([Vendetta](https://github.com/catppuccin/vendetta))
- Use older versions of discord ([Aliucord](https://github.com/catppuccin/aliucord))
- Unstable, buggy and sometimes straight up viruses 
This pretty much results in catppuccin getting near-duplicate ports but on separate repositories

So what I propose is for those repos to get moved into their own section on this repository, which already deals with desktop discord ports that go EOL. 

@sgoudham and I have already talked about this, so now it's about time to ask the community maintainers about going through with this.

**Comment by riivx** [comment]
aliucord repo is unmaintained, but not sure about merging vendetta and pc discord

**Comment by riivx** [comment]
might be a good idea, we just need to plan everything beforehand

**Comment by ToxicAven** [comment]
With discord mobile mods we have repos for going eol, I feel it would be a better idea to archive those and if/when the magical discord mod that exists for more than like 6 months rolls around, we house it here instead of it's own repo.

Aliucord is still a functional client, but if the port is complete and aliucord has no plans of breaking it (the whole point is to not break it) I think letting each repo run it's course will be what ends up happening. 

Currently the only mobile client mod (for Android at least, can't speak on iOS) that looks promising is pyoncord, of which the developer is on hiatus.

Speaking of hiatus, I have around one month of school left, and after that I am looking forward to making a big return to this repo and the catppuccin ecosystem, and I'll likely be doing some small maintenance on ctp/discord in the coming days as I find the free time.

Tl;dr either leave them or archive them, merging them makes little sense, the mobile ecosystem doesn't look promising, I'll be back soon

**Comment by Moodzz1** [comment]
Regarding pyoncord/bunny - The developer is making progress and working on it iirc, the project is alive and plugin Devs and theme Devs are doing their things.

As for Aliucord, I can't imagine the themes breaking due to aliucord sticking to the good old 126 version of discord. I'm not sure as to whether aliucord updates themselves can cause the themes to break but I don't see why that would happen ¯\_(ツ)_/¯.
> Creating aliucord themes is not hard from my experiences so if the current themes are broken they can be fixed and shouldn't require (much?) maintaining.

Unbound (aka enmity V3) is also around the corner so themes for that will also be made (converting enmity/vendetta themes to unbound does not seem hard).

If we are going to go through with this then it will only make sense, imo, to include 
- Aliucord
- Pyoncord/bunny & enmity (they can go together since they use the same theme format)
- Unbound (when that releases). 

I personally think we should go for it because we already have 2 separate repos for Vendetta and enmity despite them being the same theme (the enmity owns are incredibly out of date and not maintained). We don't want another one for pyoncord surely lol.

**Comment by sgoudham** [comment]
This is no longer planned/relevant as https://github.com/catppuccin/vendetta/pull/36 merged all of the mobile repository formats (Enmity, Bunny, Revenge) into [catppuccin/vendetta](https://github.com/catppuccin/vendetta). I believe our Aliucord repository has been revived as well.

Thanks everyone for their thoughts and opinions in this thread, I'll be closing this issue now.
